test/README.md

```diff
@@ -0,0 +1,105 @@
+# Tests
+
+###abort
+
+Tests for when the `--abort-on-uncaught-exception` flag is used.
+
+| Runs on CI   |
+|:------------:|
+| No           |
+
+###addons 
+
+Tests for [addon](https://nodejs.org/api/addons.html) functionality.
+
+| Runs on CI   |
+|:------------:|
+| Yes          |
+
+###cctest 
+
+**TODO**
+
+| Runs on CI   |
+|:------------:|
+| No           |
```

Comment:
Yeah, it's run as part of `make`. If `git blame` isn't misleading me, @bnoordhuis wrote the one file in that directory and may be able to supply a half-sentence summary of what it does if no one else steps up. If all else fails, you can just say something hand-waving like "C++ test that is run as part of the build process".